app/models/post.rb

```diff
@@ -1,3 +1,6 @@
 class Post < ApplicationRecord
+  has_one_attached :post_image
```

Comment:
Not familiar with active storage so may be a dumb q - but does this line mean there _has_ to be an image attached? What about if someone wanted to make a post with multiple images - is this much more complicated?

Reply:
In this case you would write `has many attached` to be able to upload multiple files - but it does not error if none attached